File: app/src/main/java/can/siempredelao/f1kotlin/backend/MRDataRaces.java
package can.siempredelao.f1kotlin.backend;

import com.google.gson.annotations.SerializedName;

public class MRDataRaces {

    public String xmlns;
    public String series;
    public String url;
    public String limit;
    public String offset;
    public String total;
    @SerializedName("RaceTable")
    public RaceTable raceTable;

}

File: app/src/main/kotlin/can/siempredelao/f1kotlin/backend/model/MRDataRaces.kt
package can.siempredelao.f1kotlin.backend.model

import com.google.gson.annotations.SerializedName

class MRDataRaces(val xmlns: String,
                  val series: String,
                  val url: String,
                  val limit: String,
                  val offset: String,
                  val total: String,
                  @SerializedName("RaceTable") val raceTable: RaceTable)
